src/add_prefix.rs

```diff
@@ -52,6 +53,21 @@ where
     }
 }
 
+fn add_prefix_on_vehicle_journey_ids(
+    vehicle_journey_ids: &HashMap<(String, u32), String>,
```

Comment:
Is it possible to take a `&mut` instead of reallocating entirely a new `HashMap` ?

Reply:
Hmm, the key is modified so it might not be possible.